src/wiki/plugins/images/templatetags/wiki_thumbnails.py

```diff
@@ -0,0 +1,2 @@
+from __future__ import absolute_import
```

Comment:
As of now, it's needed because a lot of HTML files say `{% load wiki_thumbnails %}` but I don't know if adding the `register` from sorl actually makes any difference.. I should try deleting it, you're right!